lib/soapy_cake/client.rb

```diff
@@ -22,7 +22,7 @@ def xml_response?
     attr_reader :domain, :api_key, :time_converter, :opts, :logger
 
     def run(request)
```

Comment:
Yes, it is. But we still use cake-issues branch in BO. So, I've back-merged master to keep this PR mergeable with the cake-issues branch.